packages/data/package.json

```diff
@@ -29,6 +29,7 @@
   },
   "dependencies": {
     "@miragejs/graphql": "^0.1.8",
+    "@rsql/parser": "^1.0.2",
```

Comment:
We could probably add something similar in the build for this
https://github.com/yahoo/navi/blob/master/packages/data/index.js

Reply:
Are you talking about removing it from the prod build? Since we remove the mirage directory for prod builds, auto import never brings it in. I validated this prior to the PR.